### ISSUE TITLE
Pay with paypal

### ISSUE THREAD

**Issue by vivekraghuram** [calblueprint/revolv#70]
Allow users to pay with their existing paypal accounts so they don't have to enter in credit card info.

**Comment by noahsark769** [comment]
@anthonysutardja can you give me a rundown of the status of this? how far did we actually get? is there documentation anywhere on how to charge a test account?

**Comment by noahsark769** [comment]
Note to self: check and make sure this works

**Comment by andylouisqin** [comment]
Is this the same as #165?

**Comment by noahsark769** [comment]
Nope, this one references paying with a paypal account (rather than a credit card) while #165 references actually charging credit cards via the paypal credit card validation API
